src/test/java/org/openstreetmap/atlas/geography/MultiPolyLineTest.java

```diff
@@ -82,13 +82,13 @@ public void testCreateMultiLineStringFromMultiPolyLine()
     @Test
     public void testCreateMultiPolyLineFromPolyLine()
     {
-        final PolyLine polyLine1 = PolyLine.wkt(
-                "LINESTRING (10.5553105 48.3419094, 10.5552096 48.3417501, 10.5551312 48.3416583, "
+        final PolyLine polyLine1 = PolyLine
+                .wkt("LINESTRING (10.5553105 48.3419094, 10.5552096 48.3417501, 10.5551312 48.3416583, "
```

Comment:
Gradle spotlessApply formatting. No number changes.

Reply:
Wow how come spotless did not catch that before!

Reply:
Created another PR in the mean time to fix that. It should seamlessly merge with your change.